common_benchmark_suite/openxla/benchmark/def_types.py

```diff
@@ -42,7 +42,8 @@ class ModelArtifactType(Enum):
   TF_SAVEDMODEL_V1 = "tf_savedmodel_v1"
   TF_SAVEDMODEL_V2 = "tf_savedmodel_v2"
   XLA_HLO_DUMP = "xla_hlo_dump"
-  STABLEHLO = "stablehlo"
+  STABLEHLO_MLIR = "stablehlo_mlir"
+  LINALG_MLIR = "linalg_mlir"
```

Comment:
Same as XLA HLO, it's a little bit questionable if Linalg MLIR can be considered as a stable format. I guess it's fine for now.

Reply:
We've been running the same linalg mlir files in IREE for the last 6 months without any issues. So far so stable.